crates/transaction-emitter-lib/src/emit.rs

```diff
@@ -642,21 +622,58 @@ impl<'t> TxnEmitter<'t> {
         let stop = Arc::new(AtomicBool::new(false));
         let stats = Arc::new(StatsAccumulator::default());
         let tokio_handle = Handle::current();
+        let mut creator_account = LocalAccount::generate(&mut self.rng);
+        let collection_name = "collection name".to_owned().into_bytes();
+        let token_name = "token name".to_owned().into_bytes();
+        if let TransactionType::NftMint = req.transaction_type {
+            initialize_nft_collection(
```

Comment:
we probably can define some init function for each payload type, I imagine p2p/mint needs to have account creations too